src/kili/presentation/client/issue.py

```diff
@@ -209,3 +209,16 @@ def issues(
         if as_generator:
             return issues_gen
         return list(issues_gen)
+
+    def update_issue_status(self, issue_id: IssueId, status: IssueStatus) -> Dict[str, Any]:
```

Comment:
```suggestion
    def update_issue_status(self, issue_id: str, status: IssueStatus) -> Dict[str, Any]:
```

Reply:
@Jonas1312 I guess it's the same for IssueStatus then?

Reply:
not it works fine with Literal types

![image](https://github.com/kili-technology/kili-python-sdk/assets/15847892/bff330e1-1bc9-47f9-8645-4351bcc6493d)

You can render the documentation using `mkdocs serve`, and check that your user-facing method accepts native python types.

The user shouldn't have to import anything from `kili.domain.***`